### PR TITLE
add User from DB auth

### PR DESCRIPTION
**Pull request by KovalAlexey**

### CODE REVIEW

File: src/main/java/com/dev/cinema/config/AppConfig.java
Comment (RomanDubovskyi) [inline src/main/java/com/dev/cinema/config/AppConfig.java]: don't add this empty line, it's redundant

File: src/main/java/com/dev/cinema/config/SecurityConfig.java
Comment (RomanDubovskyi) [inline src/main/java/com/dev/cinema/config/SecurityConfig.java]: it's better to use constructor injection

File: src/main/java/com/dev/cinema/controllers/InjectData.java
Comment (RomanDubovskyi) [inline src/main/java/com/dev/cinema/controllers/InjectData.java]:
```suggestion
    private static final String ADMIN_ROLE_NAME = "ADMIN";
```

File: src/main/java/com/dev/cinema/controllers/InjectData.java
Comment (RomanDubovskyi) [inline src/main/java/com/dev/cinema/controllers/InjectData.java]: same

File: src/main/java/com/dev/cinema/controllers/InjectData.java
Comment (RomanDubovskyi) [inline src/main/java/com/dev/cinema/controllers/InjectData.java]: cineamHall ?

File: src/main/java/com/dev/cinema/controllers/InjectData.java
Comment (RomanDubovskyi) [inline src/main/java/com/dev/cinema/controllers/InjectData.java]: do you add only admin here?

File: src/main/java/com/dev/cinema/dao/impl/RoleDaoImpl.java
Comment (RomanDubovskyi) [inline src/main/java/com/dev/cinema/dao/impl/RoleDaoImpl.java]: logger declaration

File: src/main/java/com/dev/cinema/security/CustomUserDetailsService.java
Comment (RomanDubovskyi) [inline src/main/java/com/dev/cinema/security/CustomUserDetailsService.java]: use orElseThrow with UsernameNotFoundException here

**Review by RomanDubovskyi — APPROVED** [review]
fix comments and you can merge PR without second review

File: src/main/java/com/dev/cinema/config/SecurityConfig.java
Comment (KovalAlexey) [reply src/main/java/com/dev/cinema/config/SecurityConfig.java]: done

File: src/main/java/com/dev/cinema/controllers/InjectData.java
Comment (KovalAlexey) [reply src/main/java/com/dev/cinema/controllers/InjectData.java]: done

File: src/main/java/com/dev/cinema/controllers/InjectData.java
Comment (KovalAlexey) [reply src/main/java/com/dev/cinema/controllers/InjectData.java]: done

File: src/main/java/com/dev/cinema/controllers/InjectData.java
Comment (KovalAlexey) [reply src/main/java/com/dev/cinema/controllers/InjectData.java]: done

File: src/main/java/com/dev/cinema/controllers/InjectData.java
Comment (KovalAlexey) [reply src/main/java/com/dev/cinema/controllers/InjectData.java]: done

File: src/main/java/com/dev/cinema/dao/impl/RoleDaoImpl.java
Comment (KovalAlexey) [reply src/main/java/com/dev/cinema/dao/impl/RoleDaoImpl.java]: done

File: src/main/java/com/dev/cinema/security/CustomUserDetailsService.java
Comment (KovalAlexey) [reply src/main/java/com/dev/cinema/security/CustomUserDetailsService.java]: done

File: src/main/java/com/dev/cinema/config/AppConfig.java
Comment (KovalAlexey) [reply src/main/java/com/dev/cinema/config/AppConfig.java]: done

**Pull request merged**
